### PR TITLE
 WIP: Start some work on sound scaling (fixes #2393)

### LINKED ISSUE #2393: Scaling sound using alpha for other tees
While images of other tees are scaling using alpha their sounds are still loud. These sounds could be scaled too by this coefficient

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

**Comment by def-** [comment]
This is not possible for example for jump, attach ground, noattach sounds at the moment, since CGameContext::CreateSound will send a CNetEvent_SoundWorld without the information which client caused the event.

Not sure if it makes sense to implement for the other sounds then.

A solution would be to implement a new SoundWorld2 event with the client id information that is only sent to new DDNet clients. This all seems like a lot of work for little benefit.

File: src/game/client/gameclient.cpp [hunk 1]
```diff
@@ -1563,7 +1563,7 @@ void CGameClient::OnNewSnapshot()
 				vec2 Pos = mix(vec2(m_Snap.m_aCharacters[i].m_Prev.m_X, m_Snap.m_aCharacters[i].m_Prev.m_Y),
 						vec2(m_Snap.m_aCharacters[i].m_Cur.m_X, m_Snap.m_aCharacters[i].m_Cur.m_Y),
 						Client()->IntraGameTick(g_Config.m_ClDummy));
-				m_pEffects->AirJump(Pos);
+				m_pEffects->AirJump(Pos, IsOtherTeam(i) ? g_Config.m_ClShowOthersAlpha / 100.0f : 1.0f);
```
Comment (heinrich5991) [inline src/game/client/gameclient.cpp:1566]: So the goal of this PR is to play other team's sounds with lower volume? And it's (currently) stopped by the fact that client cannot associate the `NetEvent`s with the right team?

**Comment by def-** [comment]
Yes

File: src/game/client/gameclient.cpp (hunk 1 above)
Comment (def-) [reply src/game/client/gameclient.cpp:1566]: Yes

**Pull request closed without merging**